cartography/data/indexes.cypher

```diff
@@ -42,6 +42,7 @@ CREATE INDEX ON :DOProject(id);
 CREATE INDEX ON :DynamoDBGlobalSecondaryIndex(id);
 CREATE INDEX ON :DynamoDBTable(arn);
 CREATE INDEX ON :DynamoDBTable(id);
+CREATE INDEX ON :EC2Images(id);
```

Comment:
We should rename this to EC2Image without the `s`

Reply:
Make necessary adjustments in other files accordingly

Reply:
Done